lib/samples/query_feature_table/query_feature_table_sample.dart

```diff
@@ -50,42 +55,50 @@ class _QueryFeatureTableSampleState extends State<QueryFeatureTableSample> {
   Widget build(BuildContext context) {
     return Scaffold(
       resizeToAvoidBottomInset: false,
-      body: Column(
-        children: [
-          TextField(
-            focusNode: _searchFocusNode,
-            controller: _textEditingController,
-            decoration: InputDecoration(
-              hintText: 'Enter a State...',
-              prefixIcon: const Icon(Icons.search),
-              suffixIcon: IconButton(
-                onPressed: dismissSearch,
-                icon: const Icon(Icons.clear),
+      // create a column with a text field and a map view.
+      body: SafeArea(
+        child: Column(
+          children: [
+            // create a text field for searching states.
+            TextField(
+              focusNode: _searchFocusNode,
+              controller: _textEditingController,
+              decoration: InputDecoration(
+                hintText: 'Enter a State...',
+                prefixIcon: const Icon(Icons.search),
+                suffixIcon: IconButton(
+                  onPressed: dismissSearch,
+                  icon: const Icon(Icons.clear),
+                ),
               ),
+              onSubmitted: onSearchSubmitted,
             ),
-            onSubmitted: onSearchSubmitted,
-          ),
-          Expanded(
-            child: ArcGISMapView(
-              controllerProvider: () => _mapViewController,
-              onMapViewReady: onMapViewReady,
+            // add a map view to the widget tree and set a controller.
+            Expanded(
+              child: ArcGISMapView(
+                controllerProvider: () => _mapViewController,
+                onMapViewReady: onMapViewReady,
+              ),
             ),
-          ),
-        ],
+          ],
+        ),
       ),
     );
   }
 
   void onMapViewReady() async {
+    // create a map with the topographic basemap style and set to the map view.
     final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
 
+    // create a feature table and a feature layer.
     _featureTable = ServiceFeatureTable.withUri(Uri.parse(
         'https://services.arcgis.com/jIL9msH9OI208GCb/arcgis/rest/services/USA_Daytime_Population_2016/FeatureServer/0'));
     await _featureTable.load();
-    _featureLayer = FeatureLayer.withFeatureTable(_featureTable);
-    _featureLayer.opacity = 0.8;
-    _featureLayer.maxScale = 10000;
+    _featureLayer = FeatureLayer.withFeatureTable(_featureTable)
+      ..opacity = 0.8
+      ..maxScale = 10000;
```

Comment:
```suggestion
    // create a feature layer and amend the opacity and max scale properties.
    _featureLayer = FeatureLayer.withFeatureTable(_featureTable)
      ..opacity = 0.8
      ..maxScale = 10000;
```
I think this can help with emphasising what the cascade notation is doing, incase a developer was not familiar with it.